src/configure/configurers/azurePipelineConfigurer.ts

```diff
@@ -0,0 +1,200 @@
+import * as path from 'path';
+import Q = require('q');
+import * as utils from 'util';
+import * as vscode from 'vscode';
+import { Configurer } from "./configurerBase";
+import * as constants from '../resources/constants';
+import { AzureDevOpsClient } from "../clients/devOps/azureDevOpsClient";
+import { generateDevOpsOrganizationName } from '../helper/commonHelper';
+import { AzureDevOpsHelper } from "../helper/devOps/azureDevOpsHelper";
+import { TargetResourceType, WizardInputs, AzureSession, RepositoryProvider } from "../model/models";
+import { ServiceConnectionHelper } from '../helper/devOps/serviceConnectionHelper';
+import { Messages } from '../resources/messages';
+import { GraphHelper } from '../helper/graphHelper';
+import { telemetryHelper } from '../helper/telemetryHelper';
+import { TracePoints } from '../resources/tracePoints';
+import { UniqueResourceNameSuffix } from '../configure';
+import { AppServiceClient } from '../clients/azure/appServiceClient';
+import { AzureResourceClient } from '../clients/azure/azureResourceClient';
+import { TelemetryKeys } from '../resources/telemetryKeys';
+import { Build } from '../model/azureDevOps';
+import { LocalGitRepoHelper } from '../helper/LocalGitRepoHelper';
+import { ControlProvider } from '../helper/controlProvider';
+
+const Layer = 'AzurePipelineConfigurer';
+
+export class AzurePipelineConfigurer implements Configurer {
+    private azureDevOpsHelper: AzureDevOpsHelper;
+    private azureDevOpsClient: AzureDevOpsClient;
+    private queuedPipeline: Build;
+
+    constructor(azureSession: AzureSession) {
+        this.azureDevOpsClient = new AzureDevOpsClient(azureSession.credentials);
+        this.azureDevOpsHelper = new AzureDevOpsHelper(this.azureDevOpsClient);
+    }
+
+    public async getInputs(inputs: WizardInputs): Promise<void> {
+        try {
+            if (inputs.sourceRepository.repositoryProvider === RepositoryProvider.AzureRepos) {
```

Comment:
This will always be true here, do you want to remove the else part?

Reply:
I kept it as it might be used in no-remote scenario ?